localstack/services/s3/provider.py

```diff
@@ -178,6 +271,7 @@ def _bucket_response_head(fn, self, bucket_name, *args, **kwargs):
         code, headers, body = fn(self, bucket_name, *args, **kwargs)
         bucket = self.backend.get_bucket(bucket_name)
         headers["x-amz-bucket-region"] = bucket.region_name
+        headers["content-type"] = "application/xml"
         return code, headers, body
```

Comment:
maybe this is out of scope, but would it be better to remove this patch all together, and wrap the `head_bucket` operation? or is that not an option because the moto response is a problem for our parser?

Reply:
Talked to @bentsku about it: we probably change it in a future PR.